src/components/Members Section/MemberCard.tsx

```diff
@@ -0,0 +1,71 @@
+import Image from 'next/image'
+import Link from 'next/link'
+
+type MemberProps = {
+  user: {
+    id: number,
+    image: string,
+    name: string,
+    twitter?: string,
+    linkedin?: string,
+    instagram?: string,
+    github?: string,
+  }
+}

```

Comment:
can you move this to `membersSection.type.tsx`?

Reply:
Ok. Misspelled it by mistake